File: C++/queue.cpp
#include <iostream>
#include <queue>
using namespace std;

int main() {
    queue<int> q;

    // Enqueue elements
    q.push(1);  // Queue: [1]
    q.push(2);  // Queue: [1, 2]
    q.push(3);  // Queue: [1, 2, 3]

    cout << "Front: " << q.front() << endl;  // Output: 1

    q.pop();  // Remove 1
    cout << "After pop, front: " << q.front() << endl;  // Output: 2
}

File: Java/queue.java
import java.util.*;

public class QueueExample {

    public static void main(String[] args) {
        // Create a Queue using LinkedList
        Queue<Integer> queue = new LinkedList<>();

        // Enqueue elements
        queue.add(1);  // Add 1
        queue.add(2);  // Add 2
        queue.add(3);  // Add 3

        System.out.println("Queue after adding: " + queue);  // [1, 2, 3]

        // Dequeue element
        int removed = queue.remove();  // Removes 1 (First in, first out)
        System.out.println("Removed: " + removed);  // 1
        System.out.println("Queue now: " + queue);  // [2, 3]
    }
}
